packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -290,4 +293,17 @@ contract TaikoL1 is EssentialContract, IHeaderSync, TaikoEvents {
     function getConfig() public pure virtual returns (TaikoData.Config memory) {
         return LibSharedConfig.getConfig();
     }
+
+    function isBlockVerifiable(
+        uint256 blockId,
+        bytes32 parentHash
+    ) public view returns (bool) {
```

Comment:
added helper method for testing, not sure if we need to keep it, but could be handy for clients